testhelper/docker/resource/transformer/transformer.go

```diff
@@ -0,0 +1,100 @@
+package transformer
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+
+	"github.com/rudderlabs/rudder-go-kit/httputil"
+	"github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource"
+)
+
+type Resource struct {
+	TransformerURL string
+	Port           string
+}
+
+type config struct {
+	repository   string
+	tag          string
+	exposedPorts []string
+	env          []string
+}
+
+func WithConfigBackendURL(url string) func(*config) {
+	return func(conf *config) {
+		conf.env = []string{fmt.Sprintf("CONFIG_BACKEND_URL=%s", url)}
+	}
```

Comment:
Wouldn't it be better to do something like:

```go
conf.env = append(conf.env, fmt.Sprintf("CONFIG_BACKEND_URL=%s", url))
```

Assuming another config needs to modify the env slice

Reply:
yeah, PR is merged now, will pick this later
https://linear.app/rudderstack/issue/PIPE-791/appendupdate-env-variable-while-changing-config-be-url